addons/mail/static/src/components/chat_window_manager/chat_window_manager_tests.js

```diff
@@ -553,6 +553,56 @@ QUnit.test('chat window: open / close', async function (assert) {
     assert.verifySteps(['rpc:channel_fold/open']);
 });
 
+QUnit.test('chat window: focus on ESCAPE', async function (assert) {
+    assert.expect(1);
+
+    this.data['res.partner'].records = [{
+        email: "testpartnert@odoo.com",
+        id: 11,
+        name: "TestPartner",
+    }];
+    Object.assign(this.data.initMessaging, {
+        channel_slots: {
+            channel_channel: [
+                {
+                    channel_type: 'channel',
+                    id: 20,
+                    is_minimized: true,
+                    is_pinned: true,
+                    name: "General",
+                    state: 'open',
+                    fold_state: 'open',
```

Comment:
not necessary, `fold_state` is the server variable but it is transformed into just `state` during formatter `channel_info` when sent to js.